extras/courses.md

```diff
@@ -23,6 +26,11 @@ Courses | Duration | Effort
 
 Courses | Duration | Effort
 :-- | :--: | :--:
+
+[Calculus Applied](https://www.edx.org/course/calculus-applied-harvardx-calcapl1x) | 7-9 weeks | 3-6 hours/week
```

Comment:
I can't find any reviews for this course, and it also doesn't seem terribly relevant to computer science.